packages/registry/openapi3.yaml

```diff
@@ -0,0 +1,157 @@
+openapi: 3.0.1
+info:
+  title: registry
+  description: api for managing namespaces and services in the vector ecosystem
+  version: 1.0.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+tags:
+  - name: service
+    description: a service in the vector ecosystem
+paths:
+  /service/{serviceId}:
+    get:
+      operationId: getService
+      tags:
+        - service
+      summary: get detailed info on the service
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/service'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        404:
+          $ref: '#/components/responses/NotFound'
+        5XX:
+          $ref: '#/components/responses/UnexpectedError'
+  /service/{serviceId}/rotate:
+    patch:
+      operationId: patchRotation
+      tags:
+        - service
+      summary: bump rotation for a service and its children
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+      responses:
+        201:
```

Comment:
since you have no response why the status code is not 204?

Would you like to return the new rotation to the client?

Reply:
204 it is, there is no need to return it.